src/Pages/Calculator/QuestionScreen.jsx

```diff
@@ -78,7 +78,19 @@ const QuestionScreen = props => {
             </Grid>
             <Grid container justifyContent="center" className={classes.buttonGrid}>
                 {props.buttons.map((b, idx) => {
-                    return <PrimaryButton href={b.href} key={idx} text={b.text} className={classes.button} />;
+                    return (
+                        <PrimaryButton
+                            href={b.href}
+                            key={idx}
+                            text={b.text}
+                            className={classes.button}
+                            header={props.header}
+                            body={props.body}
+                            onButtonClick={props.onButtonClick}
+                            origin="QuestionScreen"
```

Comment:
Could we use a custom type here for `origin` prop instead of a raw string? For example, in our file `calculatorPageTypes.ts` we have created custom `enum`s to represent different variables. So we could something like: 
```
export enum OriginType {
    QUESTION_SCREEN = "QuestionSreen",
    SPECIAL_CASE = "SpecialCaseTitle"
}
```
etc